src/units/abilities.cpp

```diff
@@ -182,34 +182,52 @@ bool affects_side(const config& cfg, std::size_t side, std::size_t other_side)
 		return cfg["affect_allies"].to_bool();
 }
 
+	// This function defines in which direction loc is relative to from_loc
+	// either by pointing to the hexagon loc is on or by pointing to the hexagon adjacent to from_loc closest to loc.
+	int find_direction(const map_location& loc, const map_location& from_loc, unsigned int distance)
+	{
+		const auto adjacent = get_adjacent_tiles(from_loc);
+		for(unsigned j = 0; j < adjacent.size(); ++j) {
```

Comment:
Not the it really matters in this particular case, but `size()` is type `std::size_t`, not `unsigned`.

Reply:
replaced by std::size_t